### ISSUE TITLE
`buneary create --help` displays probably wrong description for command `binding`

### ISSUE THREAD

**Issue by fyradir** [dominikbraun/buneary#2]
Current:
```
Available Commands:
  binding     Create a new queue <--
  exchange    Create a new exchange
  queue       Create a new queue
```

Expected:
```
Available Commands:
  binding     Create a new binding <--
  exchange    Create a new exchange
  queue       Create a new queue
```

**Comment by dominikbraun** [comment]
Absolutely! Thanks for opening this issue, I'm going to fix the description in an upcoming PR.

**Comment by dominikbraun** [comment]
This fix is contained in [buneary v0.1.1](https://github.com/dominikbraun/buneary/releases/tag/v0.1.1).
